cmd/admin-router.go

```diff
@@ -177,6 +186,15 @@ func registerAdminRouter(router *mux.Router, enableConfigOps bool) {
 		// Set Group Status
 		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/set-group-status").HandlerFunc(gz(httpTraceHdrs(adminAPI.SetGroupStatus))).Queries("group", "{group:.*}").Queries("status", "{status:.*}")
 
+		// Attach policies to group
+		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/attach-group-policies").HandlerFunc(gz(httpTraceHdrs(adminAPI.AttachPolicyToGroup))).Queries("group", "{group:.*}", "policiesToAdd", "{policiesToAdd:.*}")
+
+		// Detach policies from group
+		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/detach-group-policies").HandlerFunc(gz(httpTraceHdrs(adminAPI.DetachPolicyFromGroup))).Queries("group", "{group:.*}", "policiesToDetach", "{policiesToDetach:.*}")
+
```

Comment:
Same reasoning for detach policy as well. We want to show an error that the given policy is not attached (if that is the case), so this also should be a POST request. Same comments for the policy query param here.